deposit/src/main/java/edu/unc/lib/deposit/validate/VirusScanJob.java

```diff
@@ -79,35 +79,38 @@ public void runJob() {
 
         for (Entry<PID, String> href : hrefs) {
             verifyRunning();
-
-            URI manifestURI = URI.create(href.getValue());
-
-            File file = new File(manifestURI);
-
-            ScanResult result = clamScan.scan(file);
-
-            switch (result.getStatus()) {
-            case FAILED:
-                failures.put(manifestURI.toString(), result.getSignature());
-                break;
-            case ERROR:
-                throw new Error(
-                        "Virus checks are producing errors: "
-                                + result.getException()
+            PID objPid = href.getKey();
+
+            if (!isObjectCompleted(objPid)) {
```

Comment:
Just a style/minimizing changes thing, but instead of indenting everything, you can do an early continue
```
if (isObjectCompleted(objPid)) {
    continue;
}
```